pkg/cmd/operator/cleanupjob.go

```diff
@@ -118,11 +122,42 @@ func (o *CleanupJobOptions) Run(streams genericclioptions.IOStreams, cmd *cobra.
 	stopCh := signals.StopChannel()
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
+
 	go func() {
 		<-stopCh
 		cancel()
 	}()
 
+	var errs []error
+
+	err := o.runCleanup(ctx)
+	if err != nil {
+		errs = append(errs, fmt.Errorf("can't cleanup: %w", err))
+	}
+
+	select {
+	case <-stopCh:
+		klog.InfoS("Stopping any ongoing cleanup")
+		stopCleanupCtx, stopCleanupCtxCancel := context.WithTimeout(context.Background(), stopCleanupOperationTimeout)
+		defer stopCleanupCtxCancel()
+
+		err := o.scyllaClient.StopCleanup(stopCleanupCtx, o.NodeAddress)
+		if err != nil {
+			klog.Warningf("Failed to stop cleanup: %s", err)
+			errs = append(errs, fmt.Errorf("can't stop the cleanup: %w", err))
```

Comment:
nit: be consistent about article usage (you didn't use *the* cleanup in previous errors, so don't use it here, I think we can treat it as a proper noun)